glean-core/src/database/mod.rs

```diff
@@ -298,7 +298,7 @@ mod test {
 
     #[test]
     fn test_panicks_if_fails_dir_creation() {
-        assert!(Database::new("").is_err());
+        assert!(Database::new("!#\"'@#°ç").is_err());
```

Comment:
> Alessio shouting: "!#\"'@#°ç!"

(sorry, nothing to see here, please move on)

Reply:
Do we know these characters are invalid on all major platforms?  I think ext4 on Linux (Android) probably allows all of these.  Maybe include a `/`?  That's sure to fail everywhere, I think.